app/controllers/admissions_admin/groups_controller.rb

```diff
@@ -51,4 +51,58 @@ def reject_calls
     @group = Group.find(params[:id])
     @applicants_to_call = @group.applicants_to_call(@admission).sort_by(&:full_name)
   end
+
+  def show_unlogged_applicants
+    @admission = Admission.find(params[:admission_id])
+    @group = Group.find(params[:group_id])
+
+    @group_applicants = {}
+    @group_applicants[@group] = []
+    @group.jobs.where(admission: @admission).each do |job|
+      applicants_in_job = job.job_applications.map(&:applicant).flatten
+      unlogged_in_job = applicants_in_job.select do |a|
+        a.log_entries.empty?
+      end
+      job_and_applicants = JobAndApplicants.new(job, unlogged_in_job)
+      @group_applicants[@group].push(job_and_applicants)
+    end
```

Comment:
This functionality is duplicated and could be moved to a model where it makes sense, perhaps on the Admission-model?

For example something like this: 
`Admission.groupJobsWithUnloggedApplicants(group)` and `Admission.allGroupJobsWithUnloggedApplicants`
or it could be placed on the Group-model.
`Group.jobsWithUnloggedApplications(admission)`

Either way, I think that logic like this should reside in the models for reusability and separation of concerns

Reply:
Definitely! A bad case of becoming blind to your own code ...